src/platform/windows/events_loop.rs

```diff
@@ -57,6 +57,7 @@ use platform::platform::dpi::{
     enable_non_client_dpi_scaling,
     get_hwnd_scale_factor,
 };
+use platform::platform::icon::WinIcon;
```

Comment:
Keeping with alphabetical order, this should go after `platform::platform::event` below.